tests/integration/test_exoduslambda.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import logging
+import requests
+
+from .utils import get_config, get_db_client
+from boto3.dynamodb.conditions import Key, Attr
+
+
+class TestExodus:
+    def setup(self):
+        self.config = get_config()
+        self.db_client = get_db_client()
+
+    def test_cache_headers(self):
```

Comment:
Could we please not have this test added, and instead stick with the other one only?

Reply:
yes